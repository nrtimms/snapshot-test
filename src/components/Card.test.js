import BusinessCard from './Card'
import renderer from 'react-test-renderer'

test('renders a snapshot', () => {
  const tree = renderer.create(<BusinessCard />).toJSON()
  expect(tree).toMatchSnapshot()
})