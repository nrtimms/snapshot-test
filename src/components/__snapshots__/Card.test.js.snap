// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="card"
>
  <div
    className="card"
    style={
      Object {
        "width": "18rem",
      }
    }
  >
    <img
      className="card-img-top"
      src="bow.jpg"
    />
    <div
      className="card-body"
    >
      <div
        className="card-title h5"
      >
        Bows before Bros
      </div>
      <p
        className="card-text"
      >
        The best place to buy cheer bows.
      </p>
    </div>
  </div>
</div>
`;